lib/kubeclient/common.rb

```diff
@@ -202,6 +205,14 @@ def resource_name(entity_type)
         entity_type.pluralize.downcase
       end
 
+      def proxy_url_service(namespace, service, protocol)
```

Comment:
Also websocket support, which IIRC is supported by the kubernetes API server proxy (need to verify).